scalafix/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.1.1
```

Comment:
🎨 is it ok if we use same version as in scio?

Reply:
Probably yes. I used the g8 template scalafix provide but I should be able to change that.